src/main/java/com/project/gamestore/controllers/RAWGController.java

```diff
@@ -62,20 +62,41 @@ public void processTitle(String title) {
 
     @GetMapping("/videogames/search")
     public List<VideoGame> ListGamesByName(@RequestParam String title) throws JsonProcessingException {
-        // Process title
+        // Process title.
         processTitle(title);
 
-        // Call the RAWG API to get a certain number of games
+        String url = "https://api.rawg.io/api/games?search=" + title + "&page_size=10&key=79fc5d7fcd144b99ade6f0aafc6e8c74";
```

Comment:
Use page_size parameter and page number parameter from the request.

Also use the exclude_additions, exclude_parents, exclude_game_series parameters in the URL to filter out results that are not videogames. Your results yield additions/expansions to the actual game.